docs/dev/background-information/bazel/rust.mdx

```diff
@@ -0,0 +1,50 @@
+# Bazel for Rust
+
+Also checkout the [FAQ](/dev/background-information/bazel/faq) for common issues and solutions.
+
+## TL;DR
+
+- Commands:
+  - `CARGO_BAZEL_REPIN=1 bazel sync --only=crate_index` after changing workspace members or dependencies.
+    - Setting `CARGO_BAZEL_ISOLATED=0` can be set if repinning is too slow, if doing it frequently in local development.
+  - `CARGO_BAZEL_REPIN=<crate name> bazel sync --only=crate_index` after adding/updating a dependency.
+
+## Overview
+
+The rules interfacing Rust are named [`rules_rust`](https://github.com/bazelbuild/rules_rust/) and they provide all the plumbing to call the Rust compiler, run the tests
```

Comment:
```suggestion
The rules interfacing Rust are named [`rules_rust`](https://github.com/bazelbuild/rules_rust/) and they provide all the plumbing to call the Rust compiler, run the tests.
```

Reply:
Oh this is an unfinished sentence, gotta finish it lol

Reply:
oh whoops 🤣 ok I'll refrain from commenting now 😄